app/controllers/nonsipp/TaskListController.scala

```diff
@@ -456,7 +459,7 @@ object TaskListController {
             .onPageLoad(srn, NormalMode)
```

Comment:
Is the navigation the same for all scenarios?

Reply:
It's not specified as AC but I can route the user straight to the disposal list page when at least 1 disposal is completed - don't know if that would be another story / if we have that on the backlog somewhere or if we do it as part of this

Reply:
I was thinking about the last line of each AC: "and link navigates to...."

Reply:
woops missed that, will change